charts/geoweb-opmet-backend/templates/opmet-volume.yaml

```diff
@@ -25,5 +24,4 @@ spec:
     namespace: {{ .Release.Namespace }}
     name: {{ .Values.opmet.name }}-claim
   hostPath:
-    path: {{ .Values.opmet.customConfigurationFolderPath | quote }}
-{{- end }}
\ No newline at end of file
+    path: {{ .Values.opmet.customConfigurationMountPath | quote }}
```

Comment:
This should be `path: {{ .Values.opmet.customConfigurationFolderPath | quote }}`

Reply:
Done in latest commit